source/includes/User/user.md

```diff
@@ -19,7 +19,7 @@ The real-life person is represented by a Person record in OpenMRS, and a person
 > Get all non-retired Users 
 
 ```shell
-GET user?q=admin&v=default&limit=1
```

Comment:
What is the idea behind removing limit=1?

Reply:
If the header says "Get **all** non-retired Users" then why the default should be to return only one result?